engine/src/render/depth/plugin.cpp

```diff
@@ -0,0 +1,46 @@
+#include <cubos/core/io/window.hpp>
+#include <cubos/core/reflection/external/primitives.hpp>
+
+#include <cubos/engine/render/depth/depth.hpp>
+#include <cubos/engine/render/depth/plugin.hpp>
+#include <cubos/engine/render/target/plugin.hpp>
+#include <cubos/engine/render/target/target.hpp>
+
+using cubos::core::gl::Texture2DDesc;
+using cubos::core::gl::TextureFormat;
+using cubos::core::gl::Usage;
+using cubos::core::io::Window;
+
+CUBOS_DEFINE_TAG(cubos::engine::createRenderDepthTag);
+CUBOS_DEFINE_TAG(cubos::engine::drawToRenderDepthTag);
+
+void cubos::engine::renderDepthPlugin(Cubos& cubos)
+{
+    cubos.component<RenderDepth>();
```

Comment:
```suggestion
{
    cubos.depends(windowPlugin);
    cubos.depends(renderTargetPlugin);

    cubos.component<RenderDepth>();
```

Dependencies are missing.